CHANGELOG.md

```diff
@@ -8,6 +8,63 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Fixed
 - Fix DhtOps helper functions `getChainOpType`, `getChainOpAction`, `getChainOpEntry` and `getChainOpSignature` to adhere to the new types
 ### Changed
+- Updated types to the new [enum serialization convention](https://docs.rs/holochain_conductor_api/0.5.0-dev.19/holochain_conductor_api/docs/index.html#enum-serialization-convention). The following types are affected:
+  - `SignalType`
+  - `CellInfo`
+  - `InstallAppRequest`
+  - `RoleSettings`
+  - `DnaSource`
+  - `RegisterDnaRequest`
+  - `CoordinatorSource`
+  - `AppBundleSource`
+  - `AppStatusFilter`
+  - `CloneCellId`
+  - `GrantedFunctions`
+  - `CapAccess`
+  - `DnaStorageBlob`
+
+  Enum variants are now consistently `snake_case` and typically distinguished by a `type` attribute, unless for content-less enum variants.
```

Comment:
The official name is "unit-like" https://doc.rust-lang.org/reference/items/enumerations.html#r-items.enum.path-expr
```suggestion
  Enum variants are now consistently `snake_case` and typically distinguished by a `type` attribute, unless for unit-like enum variants.
```

Reply:
Addressed with slight adaptation because the initial sentence was already imprecise.